wear/src/main/java/io/homeassistant/companion/android/home/views/ListHeader.kt

```diff
@@ -29,12 +32,13 @@ fun ListHeader(
 }
 
 @Composable
-fun ListHeader(id: Int) {
+fun ListHeader(id: Int, topPadding: Dp = 0.dp) {
     ListHeader {
         Row {
             Text(
                 text = stringResource(id = id),
-                color = Color.White
+                color = Color.White,
+                modifier = Modifier.padding(top = topPadding)
```

Comment:
https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier
Rather than passing in a padding we should pass in the whole modifier.

Reply:
Makes more sense indeed :)